frontend/src/components/SearchResultsItem.js

```diff
@@ -294,63 +266,87 @@ const SearchResultsItem = ({
-      )}
+      <div style={prof_style} className={Styles.ellipsis_text}>
+        {course.professor_names.length === 0
+          ? 'TBA'
+          : course.professor_names.join(' • ')}
+      </div>
+
       {/* Course Meets */}
-      {ROW_WIDTH > COL_SPACING.MEET_CUT && (
-        <div style={meet_style}>
-          <div className={Styles.ellipsis_text}>{course.times_summary}</div>
-        </div>
-      )}
+
+      <div style={meet_style}>
+        <div className={Styles.ellipsis_text}>{course.times_summary}</div>
+      </div>
+
       {/* Course Location */}
-      {ROW_WIDTH > COL_SPACING.LOC_CUT && (
-        <div style={loc_style}>
-          <div className={Styles.ellipsis_text}>{course.locations_summary}</div>
-        </div>
-      )}
+
+      <div style={loc_style}>
+        <div className={Styles.ellipsis_text}>{course.locations_summary}</div>
+      </div>
+
       {/* Skills and Areas */}
-      {ROW_WIDTH > COL_SPACING.SA_CUT && (
-        <div style={sa_style} className="d-flex pr-2">
-          <span className={Styles.skills_areas + ' '}>
-            {course.skills.map((skill) => (
-              <Badge
-                variant="secondary"
-                className={Styles.tag + ' my-auto'}
-                key={key++}
-                style={{
-                  color: skillsAreasColors[skill],
-                  backgroundColor: chroma(skillsAreasColors[skill])
-                    .alpha(0.16)
-                    .css(),
-                }}
-              >
-                {skill}
-              </Badge>
-            ))}
-            {course.areas.map((area) => (
-              <Badge
-                variant="secondary"
-                className={Styles.tag + ' my-auto'}
-                key={key++}
-                style={{
-                  color: skillsAreasColors[area],
-                  backgroundColor: chroma(skillsAreasColors[area])
-                    .alpha(0.16)
-                    .css(),
-                }}
-              >
-                {area}
-              </Badge>
-            ))}
+
+      <div style={sa_style} className="d-flex">
+        <span className={Styles.skills_areas + ' '}>
+          {course.skills.map((skill) => (
+            <Badge
+              variant="secondary"
+              className={Styles.tag + ' my-auto'}
+              key={key++}
+              style={{
+                color: skillsAreasColors[skill],
+                backgroundColor: chroma(skillsAreasColors[skill])
+                  .alpha(0.16)
+                  .css(),
+              }}
+            >
+              {skill}
+            </Badge>
+          ))}
+          {course.areas.map((area) => (
+            <Badge
+              variant="secondary"
+              className={Styles.tag + ' my-auto'}
+              key={key++}
+              style={{
+                color: skillsAreasColors[area],
+                backgroundColor: chroma(skillsAreasColors[area])
+                  .alpha(0.16)
```

Comment:
A couple people mentioned that we should make these colors have even more contrast so that they really pop.

Reply:
yah changing from bezier to scale makes the colors look just like old table